docs/source/notebooks/wps_generate_climos_demo.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>Oh actually, you probably need to change this back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/thunderbird/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Good catch! </p>